.circleci/karma.circleci.conf.js

```diff
@@ -0,0 +1,19 @@
+const shared = require('../karma.conf.js')
+
+module.exports = function(karma) {
+  shared(karma)
```

Comment:
I'm not sure extending it this way will work. Is this still work in progress?

Reply:
https://circleci.com/gh/Paratii-Video/paratii-portal/1291 @bent0b0x